package/libsbp/libsbp.mk

```diff
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-LIBSBP_VERSION = v2.2.0
```

Comment:
Why do we need a different libsbp?

Reply:
There are libsbp messages this software sends that are present in this other version of libsbp that were not in v2.2.0.